timeinterval/timeinterval_test.go

```diff
@@ -434,7 +480,7 @@ func TestYamlUnmarshal(t *testing.T) {
 			t.Errorf("Error unmarshalling %s: Want %+v, got %+v", tc.in, tc.intervals, ti)
 		}
 		for _, ts := range tc.contains {
-			_t, _ := time.Parse(time.RFC822, ts)
+			_t, _ := time.Parse(time.RFC822Z, ts)
```

Comment:
I wouldn't have changed that. The purpose of those tests is to ensure non regression so if we change all the test suit we can't be sure we didn't break things. Also I don't see the benefit to use numeric timezones instead of the original short codes. I would revert this and use AEDT / AEST for the new test cases related to timezones.

Reply:
I changed these because the original format (RFC822) doesn't actually work the way I thought it did (I wrote the original tests as well). It doesn't actually support timezones like "AEDT". I realized this because it was failing on my new tests, until I read the RFC and as it turns out it only supports 4 static timezones. So I changed to this format so I could test with more timezones, including changing between daylight savings etc.

Reply:
To add some clarification to this (in case it's a blocker for merging), using RFC822 with timezones like AEDT / AEST just falls back to UTC, because it doesn't understand those time zones at all. The new format (RFC822Z) allows us to test the behaviour correctly by using numeric offsets.